examples/plot_meg_eeg_fields_3d.py

```diff
@@ -35,3 +35,12 @@
 
 # explore several points in time
 [evoked.plot_field(maps, time=time) for time in [0.09, .11]]
+
+# Compute the field maps to project MEG and EEG fields both in
+# scalp surface
+maps = make_field_map(evoked, trans_fname, subject='sample',
+                      subjects_dir=subjects_dir, n_jobs=1, meg_surf='head')
+
+# Plot both M/EEG fields in scalp surface separately
+labels = ['eeg', 'meg']
+[evoked.plot_field([maps[ind]], 0.11, labels[ind]) for ind in range(2)]
```

Comment:
No it was just making no sense for me having the M/EEG fields mixed up.

If you want them both together you can give the whole maps